test/less/declaration/2.json

```diff
@@ -0,0 +1,81 @@
+{
+  "type": "declaration",
+  "content": [
+    {
+      "type": "property",
+      "content": [
+        {
+          "type": "ident",
+          "content": "@{prop}",
```

Comment:
`@{prop}` is not an ident, it is a variable, so it should be parsed as a variable. I'll take a look at this one later.